Lib/pyclbr.py

```diff
@@ -174,17 +198,22 @@ def _readmodule(module, path, inpackage=None):
                 tokentype, meth_name, start = next(g)[0:3]
                 if tokentype != NAME:
                     continue # Syntax error
+                cur_func = None
                 if stack:
-                    cur_class = stack[-1][0]
-                    if isinstance(cur_class, Class):
-                        # it's a method
-                        cur_class._addmethod(meth_name, lineno)
-                    # else it's a nested def
+                    cur_obj = stack[-1][0]
+                    if isinstance(cur_obj, Object):
```

Comment:
>> Also, we want all def functions to be pushed on the stack, don't we?

Which ones aren't being pushed on the stack?  I see the `stack.append` under the `elif token == 'def':`, so I'm not sure what I'm missing.